docs/en/latest/admin-api.md

```diff
@@ -1078,7 +1084,7 @@ Example Configuration:
     ...
     ```
 
-    After the execution is successful, nodes will not retain the original data, and the entire update is:
+    After the executiis successful, nodes will not retain the original data, and the entire update is:
```

Comment:
is this right？ maybe `After executed successfully`？ i dont think `executiis` is the correct word

Reply:
Sorry about that, I'm not sure how that was changed, it shouldn't have been. I have change it back to how it was in the recent commit